README.md

```diff
@@ -54,13 +54,12 @@ In the root of the repository, where `requirements.txt` and `package.json` locat
 
 ```bash
 python3 -m pip install -r requirements.txt
-npm install   # cnpm install
+yarn
 ```
 
-You don't need root privilege to run `npm install`. It installs stuffs in the project directory. We recommend using Node 8.
+You don't need root privilege to run `yarn`. It installs stuffs in the project directory. We recommend using Node 10.
 
 You may want to use [cnpm](https://npm.taobao.org/) and [tuna](https://pypi.tuna.tsinghua.edu.cn/)
```

Comment:
You may remove all mentions of cnpm.

Reply:
I mean they can use `yarn config set registry https://npm.taobao.org/` (but actually it won't work because of the existing lock file :(